### PR TITLE
[SW-2787] Change license to AGPL v3

### LINKED ISSUE #5623: Change license: Apache 2.0 -> AGPL
following h2o-3

jira ticket: https://h2oai.atlassian.net/browse/SW-2787

### PR DESCRIPTION
**Pull request by krasinski**
closes #5623

### CODE REVIEW

File: py-scoring/conda/h2o_pysparkling_scoring_SUBST_SPARK_MAJOR_VERSION/meta.yaml [hunk 1]
```diff
@@ -16,5 +16,5 @@ requirements:
 
 about:
   home: https://github.com/h2oai/sparkling-water.git
-  license: Apache License Version 2.0
-  license_family: Apache
+  license: AGPL-3
```
Comment (mn-mikke) [inline py-scoring/conda/h2o_pysparkling_scoring_SUBST_SPARK_MAJOR_VERSION/meta.yaml:19]: Shouldn't this be `GNU Affero General Public License v3`? https://github.com/h2oai/h2o-3/pull/15533/files#diff-88f12fed5fbaae7ae5bb9d65df632b469aad6f69e7bcadcbfda31939b01c47c3R25

File: py/conda/h2o_pysparkling_SUBST_SPARK_MAJOR_VERSION/meta.yaml [hunk 2]
```diff
@@ -22,5 +22,5 @@ requirements:
 
 about:
   home: https://github.com/h2oai/sparkling-water.git
-  license: Apache License Version 2.0
-  license_family: Apache
+  license: AGPL-3
```
Comment (mn-mikke) [inline py/conda/h2o_pysparkling_SUBST_SPARK_MAJOR_VERSION/meta.yaml:25]: ditto

**Review by wendycwong — COMMENTED** [review]
Interested to see that you actually removed the license header from some files.  This will make the next license change easier.

Marekn/MichalM:  Please approve.

**Review by mmalohlava — CHANGES_REQUESTED** [review]
Hold on mee, please.

**Comment by mn-mikke** [comment]
We will continue with Apache 2.0 license.

**Pull request closed without merging**